src/app/features/main/main.component.html

```diff
@@ -1,45 +1,49 @@
-<nav [ngClass]="isOpened ? 'categories categories_opened' : 'categories '">
-	<button
-		mat-button
-		(click)="isOpened = false"
-		class="small-screen-element"
-	>
-		x
-	</button>
-
-	<div class="categories__content">
-		<h2>List of categories:</h2>
-		<div
-			class="category"
-			*ngFor="let category of mockedCateg"
+<div class="main-container">
+	<nav [ngClass]="isOpened ? 'categories categories_opened' : 'categories '">
```

Comment:
It will have categories class by default, and only need to append categories_opened, rewrite condition